src/Microsoft.DotNet.Wpf/cycle-breakers/PresentationFramework/PresentationFramework.cs

```diff
@@ -2562,6 +2563,7 @@ public sealed partial class DateTimeAutomationPeer : System.Windows.Automation.P
         protected override bool HasKeyboardFocusCore() { throw null; }
         protected override bool IsContentElementCore() { throw null; }
         protected override bool IsControlElementCore() { throw null; }
+        protected override bool IsDialogCore() { throw null; }
```

Comment:
This "cycle-breakers" version of PresentationFramework.cs mentions IsDialogCore 3 times, but doesn't mention GetHeadingLevelCore at all.  I suspect (hope) the 3 mentions will go away when you change the base method from abstract to virtual.  If not, we'll need to dig deeper.